Library/Homebrew/cmd/commands.rb

```diff
@@ -25,6 +25,8 @@ def commands_args
   def commands
     commands_args.parse
 
+    raise "This command does not take named arguments" if args.remaining.any?
```

Comment:
Similarly: a new exception or an argument to #parse` might be a better fit?

Reply:
⬆️